cmd/pipehub/main.go

```diff
@@ -135,3 +143,31 @@ func cmdGenerateRun(configPath, workspacePath *string) func(*cobra.Command, []st
 		}
 	}
 }
+
+func cmdVersion() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "version",
+		Short: "Version the application",
+		Long:  `Version the application server.`,
+		Run:   cmdVersionRun(gitCommit, goVersion, built, version),
+	}
+
+	cmd.Flags().StringVarP(&version, "version", "", "", "show version")
+	return &cmd
+}
+
+func cmdVersionRun(gitCommit, goVersion, built, version string) func(*cobra.Command, []string) {
+	return func(cmd *cobra.Command, args []string) {
+
```

Comment:
Actually, you don't need this empty line, could you remove it? Just to keep the same pattern as the rest of the code.

Reply:
Yes, ok, I removed this blank line.